packages/rest/src/route.ts

```diff
@@ -65,6 +81,22 @@ export function $routeReset({ program }: DecoratorContext, entity: Type, path: s
   });
 }
 
+const routeOptionsKey = Symbol();
+export function setRouteOptionsForNamespace(
```

Comment:
Where is this used from?

Reply:
It's used in the other PR I referenced, meant to be used to apply route options for an entire namespace.  Check out the `armNamespace` decorator.